index.bs

```diff
@@ -3190,13 +3231,13 @@ To <dfn>trigger event-level attribution</dfn> given an [=attribution trigger=] |
     |report|'s [=event-level report/source debug key=] is not null and |report|'s
     [=event-level report/trigger debug key=] is not null, [=queue a task=] to
     [=attempt to deliver a debug report=] with |report|.
+1. [=set/Append=] |rateLimitRecord| to the [=attribution rate-limit cache=].
```

Comment:
Let's move the step before queueing the report, similarly for aggregtable reports.